src/Features/Core/Portable/Completion/Providers/AbstractSymbolCompletionProvider.cs

```diff
@@ -263,15 +263,15 @@ public override async Task ProvideCompletionsAsync(CompletionContext completionC
         CompletionOptions options,
         CancellationToken cancellationToken)
     {
-        var relatedDocumentIds = document.GetLinkedDocumentIds();
+        var relatedDocumentIds = document.Project.Solution.GetRelatedDocumentIds(document.Id);
```

Comment:
GetLinkedDocuments just calls GetRelatedDocumentIds, but removes the doc-id it is being called on.  however, in this code, we add that doc id back in.  So this can just be sidestepped by calling the helper that gives the full list (including the doc you started on).

Reply:
Could this code just call GetRelatedDocumentIds instead and change the IsEmpty count to test if only document is returned?

Reply:
i restored this to keep the ordering constraint from before.